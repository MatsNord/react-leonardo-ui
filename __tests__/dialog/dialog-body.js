import React from 'react';
import renderer from 'react-test-renderer';

import Dialog from '../../src/dialog';

test('Dialog body renders correctly', () => {
  const tree = renderer.create(
    <Dialog.Body>Body content</Dialog.Body>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Dialog body passes props correctly', () => {
  const tree = renderer.create(
    <Dialog.Body className="my-class" data-x="x" title="Body title">
      Body title
    </Dialog.Body>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
