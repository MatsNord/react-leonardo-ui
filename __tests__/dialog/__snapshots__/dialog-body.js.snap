// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog body passes props correctly 1`] = `
<div
  className="lui-dialog__body my-class"
  data-x="x"
  title="Body title"
>
  Body title
</div>
`;

exports[`Dialog body renders correctly 1`] = `
<div
  className="lui-dialog__body"
>
  Body content
</div>
`;
